assets/js/navigation.js

```diff
@@ -52,54 +54,30 @@
 		}
 	};
 
-	// Sub-menu access from touchscreens
-		var masthead       = jQuery( '#masthead' );
-		var siteNavigation = masthead.find( '.main-navigation > div > ul' );
-
-		function toggleFocusClassTouchScreen() {
-
-			// Ensure the dropdowns close when user taps outside the site header
-			jQuery( document.body ).on( 'touchstart.storefront', function( e ) {
-				if ( ! jQuery( e.target ).closest( '.main-navigation li' ).length ) {
-					jQuery( '.main-navigation li' ).removeClass( 'focus' );
-				}
-			});
-
-			// Disables the link from working on the first touch of a menu with children
-			siteNavigation.find( '.menu-item-has-children > a, .page_item_has_children > a' )
-				.on( 'touchstart.storefront', function( e ) {
-					var el = jQuery( this ).parent( 'li' );
-
-					if ( ! el.hasClass( 'focus' ) ) {
-						e.preventDefault();
-						el.toggleClass( 'focus' );
-						el.siblings( '.focus' ).removeClass( 'focus' );
-					}
-				});
-		}
-		// Add Focus Class for parents of sub-menus
-		siteNavigation.find( 'a' ).on( 'focus.storefront blur.storefront', function() {
-			jQuery( this ).parents( '.menu-item, .page_item' ).toggleClass( 'focus' );
-		});
-
-		// Triggers toggleFocusClassTouchScreen on touchscreen devices
-		if ( 'ontouchstart' in window ) {
-			jQuery( window ).on( 'resize.storefront', toggleFocusClassTouchScreen );
-			toggleFocusClassTouchScreen();
-		}
-
-	// Add focus to cart dropdown
-	jQuery( window ).load( function() {
-		jQuery( '.site-header-cart' ).find( 'a' ).on( 'focus.storefront blur.storefront', function() {
-			jQuery( this ).parents().toggleClass( 'focus' );
+	// Add focus class to li and cart dropdown
+	requestAnimationFrame(function() {
+		document.querySelectorAll('.main-navigation a, .secondary-navigation a, .site-header-cart a').forEach(function(anchor) {
```

Comment:
I've merged the focus toggling in this method. Not clear what's the behavior we desire to have when on mobile, will need some assistance on this one.